RecoTracker/LSTCore/src/alpaka/LSTEvent.h

```diff
@@ -199,13 +197,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     PixelTripletsConst getPixelTriplets(bool sync = true);
     template <typename TDev = Device>
     PixelQuintupletsConst getPixelQuintuplets(bool sync = true);
-    const TrackCandidatesConst& getTrackCandidatesWithSelection(bool inCMSSW, bool sync);
```

Comment:
@ariostas Should all of this change similarly to the rest of the `get*` functions above?

Reply:
I'd say it should just return `TrackCandidatesConst`. The other `get*` have a more complicated signature because they can return views of different SoAs, but this one has a single type

Reply:
I missed Slava's comment above. It's fine like this

Reply:
> I'd say it should just return `TrackCandidatesConst`. The other `get*` have a more complicated signature because they can return views of different SoAs, but this one has a single type

other cases with MultiCollections are also underlying return by reference.